### PR TITLE
configured spring framework

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,11 +34,53 @@
...
+        </dependency>
+        <dependency>
+            <groupId>jstl</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.11.3</version>
```
Comment (andriylagun) [inline pom.xml]: Do we need that dependency at now?

**Review by andriylagun — APPROVED** [review]
Well done!

File: pom.xml (hunk 1 above)
Comment (tarkaiev) [reply pom.xml]: agree, removed

File: src/main/java/movie/tickets/service/impl/MovieSessionServiceImpl.java [hunk 2]
```diff
@@ -3,15 +3,18 @@
...
-import movie.tickets.lib.Service;
 import movie.tickets.model.MovieSession;
 import movie.tickets.service.MovieSessionService;
+import org.springframework.stereotype.Service;
 
 @Service
 public class MovieSessionServiceImpl implements MovieSessionService {
-    @Inject
-    private MovieSessionDao movieSessionDao;
+
```
Comment (Sofasmile) [inline src/main/java/movie/tickets/service/impl/MovieSessionServiceImpl.java:12]: let's remove empty line

File: src/main/java/movie/tickets/service/impl/ShoppingCartServiceImpl.java [hunk 3]
```diff
@@ -3,20 +3,23 @@
...
 import movie.tickets.service.ShoppingCartService;
+import org.springframework.stereotype.Service;
 
 @Service
 public class ShoppingCartServiceImpl implements ShoppingCartService {
-    @Inject
-    private ShoppingCartDao shoppingCartDao;
-    @Inject
-    private TicketDao ticketDao;
+
```
Comment (Sofasmile) [inline src/main/java/movie/tickets/service/impl/ShoppingCartServiceImpl.java:15]: here too))

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**